sprint/src/main/java/com/sprint/sprint/exception/ResourceNotFoundException.java

```diff
@@ -0,0 +1,20 @@
+package com.sprint.sprint.exception;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(value= HttpStatus.NOT_FOUND)
+public class ResourceNotFoundException extends RuntimeException
```

Comment:
Good that you included this, but you will need to use it in one of your controllers!